operator/storage_test.go

```diff
@@ -98,9 +97,6 @@ func TestSetupPrivateKey(t *testing.T) {
 			sk, err := operatorStorage.GetPrivateKey()
 			require.NoError(t, err)
 
-			if test.existKey == "" && test.passedKey == "" { // new key generated
```

Comment:
do we moved it to differ placE?

Reply:
`params.SsvConfig().OperatorPublicKey` is no longer exist and therefore this check was removed, before my change the value was populated in `SetupPrivateKey`